### PR TITLE
Run code mods for fixing Django 40 warnings

### PR DESCRIPTION
**Pull request by UsamaSadiq**
### Description
- Running codemods using `django-codemod` package to resolve `Django 40` warnings.
```
djcodemod run --removed-in 4.0 .
```

### CODE REVIEW

File: wiki/models/pluginbase.py [hunk 1]
```diff
@@ -26,6 +24,8 @@
 
 """
 
+from django.utils.translation import gettext_lazy as _
```
Comment (aht007) [inline wiki/models/pluginbase.py]: Nit: imports from `django` are ungrouped.

Comment (UsamaSadiq) [reply wiki/models/pluginbase.py]: I ran `isort` to sort the `imports` in order.

Comment (UsamaSadiq) [reply wiki/models/pluginbase.py]: I'll run it again on this file. Maybe it was missed somehow.

Comment (UsamaSadiq) [reply wiki/models/pluginbase.py]: Done.

**Review by aht007 — APPROVED** [review]

**Pull request merged**